src/forms/FormFieldMapCoordinates/MapCoordinatesDrawer/MapCoordinatesDrawer.tsx

```diff
@@ -82,11 +87,11 @@ const MapCoordinatesDrawer = (props: MapCoordinatesDrawerProps): ReactElement =>
 	 * Clear the input fields of latitude and longitude
 	 */
 	const resetLocation = async () => {
-		await modalReducer.dispatch(
+		await dispatch(

```

Comment:
When resetting the location the marker should be removed, but it's still showing in the map. Also when removing both inputs manually the rest button doesn't disappear.